### PR TITLE
Add hardware definition

### PR DESCRIPTION
**Pull request by AiliSarre**
I have included the term "hardware" in English

## Author: 

- Aili Sarre

## Language: 

- English

## Terms defined:

- hardware
- 
- 

## Editor

Assign the PR based on the language to the following person:

| Language   | Person                       |
|:----------:|:----------------------------:|

| English    | @baileythegreen, @zkamvar    |

Please remove the lines not relevant to the language submitted.

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -2872,17 +2872,25 @@
       encourages raising and catching exceptions, and in some situations, requires it.
 
 
-- slug: harmonic_mean
 
 
+
+
+- slug: harmonic_mean
```
Comment (baileythegreen) [inline glossary.yml:2879]:
```suggestion
- slug: harmonic_mean


```

Fix whitespace.

**Comment by baileythegreen** [comment]
@AiliSarre Thanks for submitting this. It looks like you accidentally deleted the `- slug: harmonic_mean` line, then tried to add it back in. The checks are failing because it has been added back in to the wrong place. There must be two blank lines between each entry, and right now there are too many before, and none after it.

I've added a suggestion that should fix it.

File: glossary.yml [hunk 2]
```diff
@@ -2875,14 +2875,20 @@
 - slug: harmonic_mean
 
 
+- slug: hardware
+  en:
+    term: "hardware"
+    acronym: "HW"
+    def: >
+      Any physical component of a computer system. Hardware can be internal, such as CPU, memory, and graphic cards, or external, such as monitors and keyboards. The hardware operates in conjunction with software to produce a functioning computer system.
```
Comment (baileythegreen) [inline glossary.yml:2883]:
```suggestion
      Any physical component of a computer system. Hardware can be internal, such as CPU, memory,
      and graphics cards; or external, such as monitors and keyboards. Hardware operates in conjunction
      with software to produce a functioning computer system.
```

Keep lines from being too wide.

File: glossary.yml [hunk 3]
```diff
@@ -2875,14 +2875,22 @@
...
 
+- slug: hardware
+  en:
+    term: "hardware"
+    acronym: "HW"
+    def: >
+      Any physical component of a computer system. Hardware can be internal, such as CPU, memory,
+      and graphics cards; or external, such as monitors and keyboards. Hardware operates in conjunction
+      with software to produce a functioning computer system.
+
```
Comment (baileythegreen) [inline glossary.yml:2886]:
```suggestion


```

Fix whitespace.

**Review by baileythegreen — APPROVED** [review]

**Comment by AiliSarre** [comment]
Thank you @baileythegreen for clarifying what the problem was,  and thank you for fixing the issues. I have never done this before. It seems that you made the suggested edits and have merged the branch? Thank you for the quick help. I will try to add another term to the glossary tomorrow, following your advice.

**Comment by baileythegreen** [comment]
@AiliSarre

Yes, I fixed the whitespace and split the definition over multiple lines to keep it from being too wide. For future PRs, you should keep this in mind.

Normally I don't just push changes myself unless they're very minor, so you may see me comment with requested changes that you should then make, if you submit more entries. If you have other questions, you can submit them as an issue; you can tag me there, if you want.

**Comment by elletjies** [comment]
@all-contributors please add @ailisarre for translation

**Pull request merged**